### PR TITLE
Major clean up of left over code

### PR DESCRIPTION
**Pull request by lwjohnst86**
Merge after #1 to #5 

There was some code that looked like it could be used for other purposes, like in a function. Others looked like they would be useful for the "Usage" section of the introduction vignette. All else I deleted (of course what I thought of as "important" was based on my own reading of the code... we can always bring back necessary code).

### CODE REVIEW

File: R/simulate-data.R [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#
+# # code to simulate data?
```
Comment (lwjohnst86) [inline R/simulate-data.R]: This code looked like it could be for simulating data, but I wasn't sure.

**Comment by ClemensWittenbecher** [comment]
Not sure what I should do here. Comment or edit the code?

**Comment by lwjohnst86** [comment]
I will fix on this end, one sec.

**Comment by lwjohnst86** [comment]
Only thing to do here is that we will need to discuss what is important code to keep in the `vignettes/netcoupler.Rmd` file. But we can deal with that as we go :)

**Pull request merged**